modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -66,3 +66,7 @@ Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft A
 5. There are a number of control loops in the system that must iterate over all objects in a given namespace as a reaction to some changes in state. Having a large number of objects of a given type in a single namespace can make those loops expensive and slow down processing given state changes. The limit assumes that the system has enough CPU, memory, and disk to satisfy the application requirements.
 6. Each service port and each service back-end has a corresponding entry in iptables. The number of back-ends of a given service impact the size of the endpoints objects, which impacts the size of data that is being sent all over the system.
 --
+[NOTE]
+====
+We do not provide direct guidance on sizing your {product-title} cluster. This is because determining if your cluster is within the supported bounds of {product-title} is based on considering all of the multi demential factors that limit the cluster scale. 
```

Comment:
IBM Style Guide has this guidance, which we (no pun intended) also follow in OCP docs:

> **we** _pronoun_
Do not use to refer to IBM in technical information, as in "we recommend".